rllib/BUILD

```diff
@@ -198,13 +197,33 @@ py_test(
 
 # DDPG
 py_test(
-    name = "learning_tests_pendulum_ddpg",
+    name = "learning_tests_pendulum_ddpg_tf",
```

Comment:
Wait, shouldn't we do the framework setting the same as the non-GPU tests? Meaning, no framework specifier and the buildkite pipeline definitions pick the correct command line options?

I understand we probably need more than one GPU BK job then. Let me know if this is what you tried to avoid. Happy to do it this way for some limited amount of time.

Reply:
We could do it the old way, but that would require 3 gpu bk runners instead of 2. I'm not opposed, just didn't want to add bk runners unnecessarily. 

I could add more runners though. With 4-5 tests on the runner, the setup for gpu was taking like 30 minutes, with 20 minutes of tests. Perhaps it was a one off, but that lead to a 50 minute test runner, which is probably at the limit of what we want in terms of test time.

Reply:
Cool, let's add another runner, then. This shouldn't be a problem.